SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -1,20 +1,205 @@
+        static let shareButtonTitleFontNumber: CGFloat = 15
+        static let shareButtonWidthNumber: CGFloat = 130
+        static let shareButtonHeightNumber: CGFloat = 25
+        static let shareButtonCornerRadiusNumber: CGFloat = 10
+    }
+    
+    // MARK: - Public Properties
+
     var body: some View {
-        VStack {
+        NavigationView {
+            VStack(spacing: Constants.spacingVStack) {
+                showroomNameText.offset(y: Constants.showroomNameTextOffset)
+                carImage
+                carPicker
+                Form {
+                    priceText
+                    enginePowerText
+                    toningText
+                    steeringWheelText
+                    salonPicker
+                    HStack {
+                        underestimateText
+                        underestimateSlier
+                    }
+                }
+                HStack(spacing: Constants.spacingButton) {
+                    buyButton
+                    shareButton
+                }
+            }
         }
+    }
+    
+    // MARK: Private Properties
+    
+    private let cars = [
+        Car(name: "Пятера",
+            imageName: "11",
+            toning: "Аквариум",
+            enginePower: 74,
+            price: 25000),
+        Car(name: "Не пешком",
+            imageName: "22",
+            toning: "Гаишники попутают",
+            enginePower: 82,
+            price: 30000),
+        Car(name: "Дедушкина",
+            imageName: "33",
+            toning: "Аквариум",
+            enginePower: 123,
+            price: 15000),
+        Car(name: "Феррари",
+            imageName: "44",
+            toning: "Аквариум",
+            enginePower: 162,
+            price: 100000)
+    ]
+    
+    @State private var selectionCarPicker = Constants.zeroNumber
+    @State private var selectionSalonPicker = Constants.zeroNumber
+    @State private var isShare = false
+    @State private var isBuy = false
+    @State private var isSteeringWheel = false
+    @State private var underestimate = Constants.underestimateNumber
+
+    private var salonTypes = Constants.salonTypes
+    
+    private var showroomNameText: some View {
+        Text(Constants.showroomNameText)
+            .bold()
+            .font(.system(size: Constants.showroomNameTextFontSize))
+            .foregroundColor(.red)
+    }
+        
+    private var buyButton: some View {
+        return Button {
+            isBuy = true
+        } label: {
+            Text(Constants.buyText).font(.system(size: Constants.buiButtonFontSize))
+                .padding()
+        }.alert(isPresented: $isBuy) {
+            Alert(title: Text(Constants.buyButtonAlertTitleText), message: Text(Constants.buiButtonAlertMessageText), primaryButton: .cancel(), secondaryButton: .default(Text(Constants.buiButtonAlertDefaultText)))
+        }.frame(width: Constants.buiButtonWidthNumber, height: Constants.buiButtonHeightNumber)
```

Comment:
```suggestion
        }
        .frame(width: Constants.buiButtonWidthNumber, height: Constants.buiButtonHeightNumber)
```
Все модифаеры с новой строки всегда